common/src/main/resources/messages/messages.properties

```diff
@@ -685,6 +685,13 @@ datalake.verticalscale.datalake.in.progress=Vertical scale Data Lake is in progr
 datalake.verticalscale.datalake.failed=Vertical scale Data Lake failed {0}
 datalake.verticalscale.failed=Vertical scale failed on Data Lake {0} please use a different instance type or retry the operation
 
+datalake.horizontalscale.validation.in.progress=Validaiton for horizontal scale Data Lake is in progress
+datalake.horizontalscale.validation.failed=Validation for horizontal scale Data lake failed. Issues: {0}
```

Comment:
Here and elsewhere I would reword `horizontal scale Data Lake` to `Data Lake horizontal scaling`